src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -1106,52 +1106,31 @@ public String getQuestionText(String feedbackQuestionId) {
             getQuestionResponseMapByRecipientTeam() {
         LinkedHashMap<String, Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>>> sortedMap =
                 new LinkedHashMap<String, Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>>>();
-        Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>> responsesForOneRecipient = null;
-        List<FeedbackResponseAttributes> responsesForOneRecipientOneQuestion = null;
 
         Collections.sort(responses, compareByTeamQuestionRecipientTeamGiver);
 
-        String recipientTeam = null;
-        String questionId = null;
-
         for (FeedbackResponseAttributes response : responses) {
-            if (recipientTeam == null ||
-                    !(getTeamNameForEmail(response.recipientEmail).equals("")
-                            ? getNameForEmail(response.recipientEmail).equals(recipientTeam)
-                            : getTeamNameForEmail(response.recipientEmail).equals(recipientTeam))) {
-                if (questionId != null && responsesForOneRecipientOneQuestion != null
-                 && responsesForOneRecipient != null) {
-                    responsesForOneRecipient.put(questions.get(questionId),
-                                                 responsesForOneRecipientOneQuestion);
-                }
-                if (recipientTeam != null && responsesForOneRecipient != null) {
-                    sortedMap.put(recipientTeam, responsesForOneRecipient);
-                }
-                responsesForOneRecipient = new LinkedHashMap<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>>();
-                recipientTeam = getTeamNameForEmail(response.recipientEmail);
-                if (recipientTeam == "") {
-                    recipientTeam = getNameForEmail(response.recipientEmail);
-                }
-                questionId = null;
+            String recipientTeam = getTeamNameForEmail(response.recipientEmail);
+            if (recipientTeam == "") {
```

Comment:
LOL this is one of the things that PMD will catch once it's in 😛 
`recipientTeam.isEmpty()`